src/PHPImageWorkshop/Core/ImageWorkshopLayer.php

```diff
@@ -1569,6 +1569,14 @@ public function save($folder, $imageName, $createFolders = true, $backgroundColo
 
             $isSaved = imagepng($image, $filename, intval($imageQuality));
 
+        } elseif ($extension == 'webp') {
+            $gdInfo = gd_info(); // Get supported image types
+	    if (!isset($gdInfo['WebP Support']) && !$gdInfo['WebP Support']) {

```

Comment:
You can simply use `function_exists('imagewebp')`, I think it will be better.

Because it's compatible with a polyfill.